store/localcachelayer/channel_layer.go

```diff
@@ -63,6 +73,13 @@ func (s LocalCacheChannelStore) InvalidateMemberCount(channelId string) {
 	}
 }
 
+func (s LocalCacheChannelStore) InvalidateChannelByName(teamId, name string) {
```

Comment:
Because this function now invalidate the cache for the whole cluster we have to update who is this  handled in the `app/web_hub.go` file. So we no longer need to emit the cluster message to invalidate this specific cache here. (If we don't do that, we end up with a loop in the cluster messages).

Reply:
@jespino  i dont follow 100%  . but i raised another PR because merge conflicts were difficult. - https://github.com/mattermost/mattermost-server/pull/13623